src/app/components/header/header.component.js

```diff
@@ -27,6 +29,11 @@ export class HeaderComponent {
         return this.routes.find(route => route.headerName && route.useAsDefault);
     }
 
+    initialRouteActivated(route) {
```

Comment:
Probably we able to get rid of this function. Two propositions:
- investigate how to use 'serialize' method in @RouteConfig;
- simply remove '/dashboard' route and refactor navigationRoutes method: 
`get navigationRoutes() {
        return this.routes.filter(route => route.headerName);
    }`